python/cuml/tsa/arima.pyx

```diff
@@ -849,11 +871,11 @@ class ARIMA(Base):
 
         cdef handle_t* handle_ = <handle_t*><size_t>self.handle.getHandle()
 
-        # TODO: don't create vs array every time!
+        # TODO: don't create pred array every time!
```

Comment:
I can’t remember if there was a gh issue for this, if there is could you add a link here?